tests/ci.rs

```diff
@@ -42,6 +42,7 @@ use routing::{Client, Data, Event, FullId, Node, PlainData, RequestContent, Requ
               ResponseMessage};
 
 const GROUP_SIZE: usize = kademlia_routing_table::GROUP_SIZE as usize;
+const QUORUM_SIZE: usize = 5;
```

Comment:
We should make that `pub` in kademlia_routing_table! (Maybe add a TODO here, so we use the constant once that's done.)